packages/slate/test/index.js

```diff
@@ -45,6 +46,31 @@ describe('slate', () => {
     const result = test(input)
     assert.deepEqual(result, output)
   })
+  // make sure with or without batchDirty, the normalize result is the same
+  const testBatchDirty = ({ module }) => {
```

Comment:
for all `insertNodes & insertFragment` test cases, run and collect the `dirtyPaths` with and without `batchDirty` option, make sure they are the same